### ISSUE TITLE
Accessibility: Search input needs a label

### ISSUE THREAD

**Issue by belcherj** [Automattic/simplenote-electron#1395]
The search input in this [component](https://github.com/Automattic/simplenote-electron/blob/develop/lib/search-field/index.jsx) does not have a label.  It needs a label that is hidden

`<input type="text" placeholder="All Notes" spellcheck="false" value="">`

### What
Add a hidden lablel to the search input

### Why
To make it accessible to screen readers

### How
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/forms/Basic_form_hints

<img width="302" alt="Screen Shot 2019-06-21 at 1 54 39 PM" src="https://user-images.githubusercontent.com/6817400/59941826-2c0c8280-942c-11e9-966d-50ab6c7c5cc9.png">

**Comment by codebykat** [comment]
Stale issue, this was actually fixed in https://github.com/Automattic/simplenote-electron/pull/1217
